benefits/core/templates/core/page.html

```diff
@@ -14,57 +14,50 @@
 
 {% block main_content %}
   <div class="container-fluid">
-    <div class="row main-row">
-      {% if page.noimage %}
-        <div class="container">
-        {% else %}
-          {% load static %}
-          <div class="col-lg-6 image"></div>
-          <div class="col-md-8 offset-md-2 col-lg-6 offset-lg-0">
-          {% endif %}
-          <div class="container content">
-            {% block container_content %}
-
-              {% block icon_title %}
-                {% if page.content_title %}
-                  {% if page.icon %}
-                    {% include "core/includes/icon-title.html" with title=page.content_title icon=page.icon %}
-                  {% else %}
-                    <h1>{{ page.content_title }}</h1>
-                  {% endif %}
-                {% endif %}
-              {% endblock icon_title %}
-
-              {% block paragraph_content %}
-                {% for p in page.paragraphs %}<p>{{ p }}</p>{% endfor %}
-              {% endblock paragraph_content %}
-
-              {% block forms %}
-                {% if page.forms %}
-                  {% for f in page.forms %}
-                    {% include "core/includes/form.html" with form=f %}
+    <div class="container row main-row">
```

Comment:
I added this `container` class here b/c otherwise the page was getting un-center-aligned. I'm sure this will be all changed in the future planned page template refactors.